autoload/lsp/hover.vim

```diff
@@ -105,8 +105,19 @@ export def HoverReply(lspserver: dict<any>, hoverResult: any, cmdmods: string):
 
   # Nothing to show
   if hoverText->empty()
-    if cmdmods !~ 'silent'
-      util.WarnMsg($'No documentation found for current keyword')
+    if &keywordprg !=# ':LspHover' && &l:keywordprg !=# &g:keywordprg
+      if cmdmods !~ 'silent'
+        util.WarnMsg($'No documentation found for current keyword; falling back to built-in.')
+      endif
+      try
+        execute 'normal! K'
+      catch /.*/
```

Comment:
I wonder if simply `:silent!` would do the job as well; maybe catching is cleaner!?

Reply:
silent sounds like a good idea. Not sure if catching is overkill here. If you still want to take your time to clean up the code, might be worth to put the PR into draft.